src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -15,148 +15,110 @@
+        wristEncoder.setPositionConversionFactor(rotationToRad);
 
         wristEncoder.setZeroOffset(offsetRad[ARM]);
         armEncoder.setZeroOffset(offsetRad[WRIST]);
 
         armPID.setTolerance(posToleranceRad[ARM], velToleranceRadPSec[ARM]);
         wristPID.setTolerance(posToleranceRad[WRIST], velToleranceRadPSec[WRIST]);
 
-        //TODO: REMOVE WHEN DONE WITH TESTING (ANY CODE REVIEWERS, PLEASE REJECT MERGES TO MASTER IF THIS IS STILL HERE)
-        SmartDashboard.putNumber("LowArmCube", GoalPos.LOW[ARM].armPos);
-        SmartDashboard.putNumber("LowWristCube", GoalPos.LOW[ARM].wristPos);
-        SmartDashboard.putNumber("MidArmCube", GoalPos.MID[ARM].armPos);
-        SmartDashboard.putNumber("MidWristCube", GoalPos.MID[ARM].wristPos);
-        SmartDashboard.putNumber("HighArmCube", GoalPos.HIGH[ARM].armPos);
-        SmartDashboard.putNumber("HighWristCube", GoalPos.HIGH[ARM].wristPos);
-        SmartDashboard.putNumber("StoredArmCube", GoalPos.STORED[ARM].armPos);
-        SmartDashboard.putNumber("StoredWristCube", GoalPos.STORED[ARM].wristPos);
-        SmartDashboard.putNumber("ShelfArmCube", GoalPos.SHELF[ARM].armPos);
-        SmartDashboard.putNumber("ShelfWristCube", GoalPos.SHELF[ARM].wristPos);
-        SmartDashboard.putNumber("SubstationArmCube", GoalPos.SUBSTATION[ARM].armPos);
-        SmartDashboard.putNumber("SubstationWristCube", GoalPos.SUBSTATION[ARM].wristPos);
-        SmartDashboard.putNumber("IntakeArmCube", GoalPos.INTAKE[ARM].armPos);
-        SmartDashboard.putNumber("IntakeWristCube", GoalPos.INTAKE[ARM].wristPos);
-
-        SmartDashboard.putNumber("LowArmCone", GoalPos.LOW[WRIST].armPos);
-        SmartDashboard.putNumber("LowWristCone", GoalPos.LOW[WRIST].wristPos);
-        SmartDashboard.putNumber("MidArmCone", GoalPos.MID[WRIST].armPos);
-        SmartDashboard.putNumber("MidWristCone", GoalPos.MID[WRIST].wristPos);
-        SmartDashboard.putNumber("HighArmCone", GoalPos.HIGH[WRIST].armPos);
-        SmartDashboard.putNumber("HighWristCone", GoalPos.HIGH[WRIST].wristPos);
-        SmartDashboard.putNumber("StoredArmCone", GoalPos.STORED[WRIST].armPos);
-        SmartDashboard.putNumber("StoredWristCone", GoalPos.STORED[WRIST].wristPos);
-        SmartDashboard.putNumber("ShelfArmCone", GoalPos.SHELF[WRIST].armPos);
-        SmartDashboard.putNumber("ShelfWristCone", GoalPos.SHELF[WRIST].wristPos);
-        SmartDashboard.putNumber("SubstationArmCone", GoalPos.SUBSTATION[WRIST].armPos);
-        SmartDashboard.putNumber("SubstationWristCone", GoalPos.SUBSTATION[WRIST].wristPos);
-        SmartDashboard.putNumber("IntakeArmCone", GoalPos.INTAKE[WRIST].armPos);
-        SmartDashboard.putNumber("IntakeWristCone", GoalPos.INTAKE[WRIST].wristPos);
-
-        SmartDashboard.putNumber("Max FF Vel", MAX_FF_VEL);
-        SmartDashboard.putNumber("Max FF Accel", MAX_FF_ACCEL);
+        putArmControlOnSmartDashboard();
     }
 
     @Override
     public void periodic() {
```

Comment:
This method should actually move the arm/wrist toward the goal position/velocity (preferably via a motion profile). Right now, it doesn't actually move the arm/wrist at all. In fact, as far as I can tell, nothing in the code would actually move the arm/wrist, because nothing calls `setArmVoltage` or `setWristVoltage`.